tsconfig.json

```diff
@@ -27,7 +27,16 @@
       "dom",
       "dom.iterable"
     ],
-    "skipLibCheck": false
+    "skipLibCheck": false,
+    "strict": false,
+    "plugins": [
+      {
+        "name": "typescript-strict-plugin",
+        "paths": [
+          "./packages/@react-spectrum/button/s",
```

Comment:
Is this the key line to this change, only check strict on the components we've updated? And when all are updated we change the package.json change above again to `"check-types": "tsc-strict",`? 

Do we need to or can we also include aria or stately paths?

Reply:
other way around, this is the only files that are checked by strict, so when we want to update a component, we add a line in here to enable checking on that package, then go and fix the issues
we can include any paths we like, it's an array, and at some point we'll do stately/aria etc as well, and then eventually remove this plugin once we've done the majority of the repo so we can finish the rest